src/puppetlabs/puppetdb/status.clj

```diff
@@ -0,0 +1,39 @@
+(ns puppetlabs.puppetdb.status
+  (:require [puppetlabs.puppetdb.scf.storage-utils :as sutils]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.mq :as mq]
+            [puppetlabs.puppetdb.schema :as pls]
+            [schema.core :as s]))
+
+(def status-details-schema {:maintenance_mode? s/Bool
```

Comment:
What do you think about calling this `foss-status-details-schema` to signal that it will be different in PE?